.github/workflows/main.yml

```diff
@@ -95,6 +80,65 @@ jobs:
     - name: Remove reference magnification data
       run: rm -r testdata/tiff_mag_2_reference/
 
+    - name: Generate Docs
+      if: matrix.python-version == '3.7'
```

Comment:
is there a specific reason for using 3.7 here? I think I would prefer to build the docs with 3.8 if there's no counter argument I'm missing.

Reply:
i switched to 3.8